xen/drivers/passthrough/arm/io-pgtable-arm.c

```diff
@@ -394,21 +498,26 @@ static int arm_lpae_map(struct io_pgtable_ops *ops, unsigned long iova,
 		return 0;
 
 	prot = arm_lpae_prot_to_pte(data, iommu_prot);
+	ptep = __map_domain_page(data->pgd);
```

Comment:
It can't fail. Both P2M code and other comsumers don't check what map_domain_page returns